.travis.yml

```diff
@@ -0,0 +1,26 @@
+os: osx
+addons:
+  homebrew:
+    packages:
+      - git
+      - pkgconfig
+      - automake
+      - autoconf
+      - libtool
+      - coreutils
+install:
+  - sudo easy_install pip
+  - sudo pip install pyblake2 pyzmq
+script:
+  - ./zcutil/build.sh
```

Comment:
Btw, if you want to use the equivalent of `-j$(nproc)` for macOS, you can use `-j$(sysctl -n hw.ncpu)`